demos-europe/demosplan-core
# F add statements similar submitters to export

**Ticket:** https://yaits.demos-deutschland.de/T34766

Description: add statements similar submitters to export

### How to review/test

add Other submitters of the same statement and after export it, they should be found in the export ( Start > Procedures > Select procedures > Statements >Details and replies >Edit )

Delete the checkbox if it doesn't apply/isn't necessary.
